package.json

```diff
@@ -32,6 +32,11 @@
       "types": "./dist/langchain/index.d.ts",
       "import": "./dist/langchain/index.js",
       "require": "./dist/langchain/index.cjs"
+    },
+    "./@langchain/core": {
+      "types": "./dist/@langchain/core/index.d.ts",
+      "import": "./dist/@langchain/core/index.js",
+      "require": "./dist/@langchain/core/index.cjs"
```

Comment:
Remove this block

Reply:
The imports in code didn't work correctly without this, e.g. the suggested imports were from the old `langchain/...`